models/interfaces.ts

```diff
@@ -112,17 +85,18 @@ export interface Rollup {
   continuous: boolean;
   delay: number | null;
   description: string;
-  dimensions: (DateHistogramItem | TermsItem | HistogramItem)[];
+  dimensions: RollupDimensionItem[];
   enabled: boolean;
   enabledTime: number | null;
-  lastUpdatedTime: number;
-  metadata_id: number | null;
+  last_updated_time: number;
+  metadata_id: string | null;
   metrics: MetricItem[];
   page_size: number;
   schedule: IntervalSchedule | CronSchedule;
-  schemaVersion: number;
+  schema_version: number;
```

Comment:
If these are used on the frontend we want them to be camelCase not snake_case. If it's a big change let's create a github issue to track the work and we can do it as a follow up PR.